### ISSUE TITLE
Disconnection from GATT server is taking too long

### ISSUE THREAD

**Issue by miguel-neto-andrade** [hbldh/bleak#1360]
* bleak version: 0.20.0
* Python version: 3.10
* Operating System: Windows 11 x64

### Description

I have developed an API to facilitate communication with a BLE device. Although the API is functioning correctly, I have observed that the disconnect method is taking too long to execute, specifically around 4 seconds. This extended duration is negatively impacting the usability of my software. I am seeking guidance to determine if this is an expected behavior and if there are any potential solutions to significantly reduce the disconnection time. It is worth mentioning that every time I send a command to the device, the aforementioned method is repeated, resulting in a minimum delay of 4 seconds for each command execution.

### What I Did

In short, bleak plays the following roles in my software.

**1. Establish connection with the device:**
```
client = BleakClient(address_or_ble_device=self.device_address)
await client.connect()
await client.start_notify(characteristic_number, callback)
```
**2. Send messages:**
```
await client.write_gatt_char(another_characteristic_nmr, message)
```
**3. Disconnect from the device:**
```
await client.stop_notify(characteristic_number)
await client.disconnect()
```

**Comment by dlech** [comment]
The Windows Bluetooth stack is complicated. I have one computer where it takes 2 minutes to disconnect from any device for some reason.

If you can share your logs, maybe we could see something, but likely the issue is that it just the way Windows works.

If we ever implement #367, then perhaps we could change things so we don't actually wait for disconnect. But the fact would still remain that Windows will remain connected to the device for some time.
